repository/stamp.go

```diff
@@ -93,4 +99,10 @@ type StampRepository interface {
 	// 存在しないスタンプがあった場合、ArgumentErrorを返します。
 	// DBによるエラーを返すことがあります。
 	ExistStamps(stampIDs []uuid.UUID) (err error)
+	//
+	// 成功した場合、(統計情報、nil)を返します。
+	// //　既に関連付けがない場合、(nil,ErrNotFound)を返します。
+	// stampIDにNILを渡した場合 (nil,ErrNilID)を返します。
+	// DBによるエラーを返すことがあります。
```

Comment:
ちょっとフォーマットに気をつける（他と揃える）とぐっと良くなります
```suggestion
	// 成功した場合、(統計情報, nil)を返します。
	//
	// 既に関連付けがない場合、(nil, ErrNotFound)を返します。
	// stampIDにnilを渡した場合、(nil, ErrNilID)を返します。
	// DBによるエラーを返すことがあります。
```

Reply:
> 既に関連付けがない場合、

というよりは「スタンプが存在しない場合、」の方が良いと思います